src/components/pages/CompanySubscriptions/CompanySubscriptionDetail.tsx

```diff
@@ -0,0 +1,176 @@
+ * Copyright (c) 2021, 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import {
+  Button,
+  StaticTable,
+  Typography,
+} from '@catena-x/portal-shared-components'
+import { useLocation, useNavigate } from 'react-router-dom'
+import {
+  useFetchAppsDataQuery,
+  useFetchDocumentByIdMutation,
+  useFetchSubscriptionAppQuery,
+} from 'features/apps/apiSlice'
+import { useTranslation } from 'react-i18next'
+import { useEffect, useState } from 'react'
+import CommonService from 'services/CommonService'
+import { Box } from '@mui/material'
+import { PAGES } from 'types/Constants'
+import './Companysubscriptions.scss'
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+import UnpublishedIcon from '@mui/icons-material/Unpublished'
+import { SubscriptionStatus } from 'features/apps/types'
+
+export default function CompanySubscriptionDetail() {
+  const navigate = useNavigate()
+  const { state } = useLocation()
+  const items = state
+  const { t } = useTranslation()
+  const appId = items.offerId ?? ''
+  const subscriptionId = items.subscriptionId ?? ''
+  const { data } = useFetchSubscriptionAppQuery({ appId, subscriptionId })
+  const fetchAppsData = useFetchAppsDataQuery(appId).data
+  const [image, setImage] = useState('')
+  const [fetchDocumentById] = useFetchDocumentByIdMutation()
+
+  const tableData = {
+    head: [
+      t('content.companySubscriptions.connector'),
+      t('content.companySubscriptions.technicalUser'),
+    ],
+    body: [
+      [data?.connectorData?.[0]?.name ?? ''],
+      [data?.technicalUserData?.[0]?.name ?? ''],
+    ],
+  }
+
+  useEffect(() => {
+    if (fetchAppsData?.leadPictureId) {
+      const id = CommonService.isValidPictureId(fetchAppsData?.leadPictureId)
+      void getImage(id)
+    }
+    // eslint-disable-next-line
+  }, [fetchAppsData])
+
+  const getImage = async (documentId: string) => {
+    try {
+      const response = await fetchDocumentById({
+        appId: fetchAppsData?.id ?? '',
+        documentId,
+      }).unwrap()
+      const file = response.data
+      setImage(URL.createObjectURL(file))
+    } catch (error) {}
+  }
```

Comment:
please avoid custom image loading because we have the <Image> in the shared components for that. Or are there any special reasons why the images have to be loaded manually here?

Reply:
@oyo Made the changes